Q_R-AXIS_RCS_AUTOPILOT.s

```diff
@@ -476,7 +476,7 @@ QERRCALC	CAE	CDUY		# Q-ERROR CALCULATION
 		TS	DAPTEMP2	# SAVE FOR RERRCALC
 		EXTEND
 		MP	M22		# (CDUZ-CDUZD)*M22 SCALED AT PI RADIANS
-		AD	DELQEROR	# KALCMANU INERFACE ERROR
+		AD	DELQEROR	# KALCMANU INTERFACE ERROR
```

Comment:
`INERFACE` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1451.jpg)